cmake/ProjectMacros.cmake

```diff
@@ -112,7 +112,7 @@ endmacro()
 
 function( ADD_SIMULATION_TEST )
   set(options ANNUAL_SIMULATION DESIGN_DAY_ONLY EXPECT_FATAL)
-  set(oneValueArgs IDF_FILE EPW_FILE)
+  set(oneValueArgs IDF_FILE EPW_FILE COST)
```

Comment:
Right. basement is why you want this.  Ok can you document the cost argument in the macro, next to the other argument descriptions?  Explaining that this argument controls simulation priority and that is useful to run basement first.